### PR TITLE
removed print and method

### PR DESCRIPTION
**Pull request by nav-nandos**
Overview:
   Refactor print statement and method.

### CODE REVIEW

File: event_handler/sources.py [hunk 1]
```diff
@@ -93,9 +93,9 @@ def simple_token_verification(token, body):
     if not token:
         raise Exception("Token is empty")
     print("i am in simple token verification ")
-    secret =  get_secret(PROJECT_NAME, "event-handler", "latest").decode()
```
Comment (matthew-green-nandos) [inline event_handler/sources.py:96]: This is done on L99

Comment (nav-nandos) [reply event_handler/sources.py:96]: yeah, i removed this one.

**Comment by matthew-green-nandos** [comment]
@nav-nandos, I don't understand this change as you seem to be decoding the secret twice, on L96 and L99 base on this PR. I think you just need to undo the print statement.

**Comment by nav-nandos** [comment]
> 

@matthew-green-nandos  that was earlier one now its removed (L96 changes is getting removed in this PR). right now decode method is activated on L99(like before).

**Review by matthew-green-nandos — APPROVED** [review]

**Pull request merged**